src/app/loans/common-resolvers/waive-interest-template.resolver.ts

```diff
@@ -0,0 +1,31 @@
+/** Angular Imports */
```

Comment:
@BLasan I have setup a common resolver for all the action buttons, in the file `loan-action-button.resolver.ts` you can add an if condition as done earlier and resolve the API request. Also instead of `getLoanInterestTemplate()` you can use `getLoanActionTemplate()` in generic way for all the commands to be used with the loan template.

Reply:
But the command will differ. In waive interest they have used the command as 'waiveinterest'

Reply:
Done. Please review

Reply:
Please remove the particular file.

Reply:
@BLasan I don't see any use of this file. Can you please remove this?